src/hypothesis/extra/django/models.py

```diff
@@ -72,6 +72,9 @@ def add_default_field_mapping(field_type, strategy):
     field_mappings()[field_type] = strategy
 
 
+DEFAULT_VALUE = object()
```

Comment:
Hypothesis has a helper for objects like this that gives them a better repr. If you look in hypothesis.utils.conventions there's the UniqueIdentifier class.